src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -159,6 +160,10 @@ export default {
 		promoGuestCheckoutEnabled: {
 			type: Boolean,
 			default: false,
+		},
+		useAsyncCheckout: {
```

Comment:
Is there more UI work needed before we can roll out? Like polling status and whatnot? I'm assuming we won't know what's happening currently after this async checkout is triggered.

Reply:
The only UI work for this initial pass will be to ensure the "loading" state is active and to show and error if it's returned during polling.